### ISSUE TITLE
Functions being displayed as nodes will get their names displayed without the last chars (Not constant)

### ISSUE THREAD

**Issue by TheOnlyArtz** [luna/luna-studio#1100]
Watch as 3 functions are missing last character in the visual part vs the textual part
![image](https://user-images.githubusercontent.com/30025874/45897224-35a28480-bddf-11e8-853d-7f3c7977f04c.png)

EDIT: it looks like the only first function is the one to not being affected.

**Comment by jaen** [comment]
Reproduction steps for me are (on 1.4.101.6):

1. create an empty project
2. paste this between imports and the `main` function:
```elm
class Test:
    TestCase:
        testField :: Int 

def test:
    "test"
```
The issue is probably with class case field parsing, since it does not seem to happen if the case has no fields.

**Comment by wdanilo** [comment]
Should be fixed in https://github.com/luna/luna/pull/293

**Comment by piotrMocz** [comment]
This is fixed in `1.5`.

**Comment by mikusp** [comment]
This is fixed now
